homeassistant/components/growatt_server/sensor.py

```diff
@@ -348,15 +344,6 @@
 
 SENSOR_TYPES = {**TOTAL_SENSOR_TYPES, **INVERTER_SENSOR_TYPES, **STORAGE_SENSOR_TYPES}
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Don't remove PLATFORM_SCHEMA. It's still being used by `setup_platform`.

Reply:
`setup_platform` is an official hook and so if someone uses YAML configuration, that method is being called. Like this:

```yaml
sensor:
  platform: growatt_server
  username: x
  password: y
```